### ISSUE TITLE
Technical Research - Comparative Analysis of Semaphore vs other privacy approaches

### ISSUE THREAD

**Issue by andyguzmaneth** [semaphore-protocol/semaphore#333]
Semaphore is one of the simplest ways to add privacy to applications. What other technologies exist and how do they fit and compare to each other?

For example:

- Semaphore groups vs Sismo Groups
- Semaphore vs Nym (mixnet)
- Semaphore vs other zkID protocols

**Comment by atomniketh** [comment]
I just got approved to be in Aztec's/zkCamp's Noir course cohort. I'm thinking of doing a comparison of how I would use Noir to do what I did with Semaphore in my StealthComms project. I can maybe create a matrix to compare Semaphore vs. Noir/Nargo if that would help.

**Comment by andyguzmaneth** [comment]
Yes!! Super helpful @atomniketh  comparing performance on different implementations of Semaphore is great, (languages, proving systems, etc) 

I think @vplasencia  has done some work with that, unsure if she did Noir though

**Comment by vplasencia** [comment]
Hey @atomniketh! That would be great. Thank you!

**Comment by newgnart** [comment]
Hey @atomniketh 
I will also be in the coming zkCamp's Noir cohort, happy to help with your idea.
@aguzmant103 I did a little writeup about Semaphore [here](https://github.com/trangnv/proof-of-membership), can also dig in other projects architecture

**Comment by vplasencia** [comment]
There is a Semaphore implementation in Noir that can be useful for your project: https://github.com/StefanVelkoski/semaphore-noir
@atomniketh @trangnv

**Comment by atomniketh** [comment]
@trangnv Look for me in the ZKCamp circles group. Let's coordinate there!

**Comment by atomniketh** [comment]
|   ZK Language   |        Noir        |      Circom     |     Cairo     |    Leo    |
|-----------------|--------------------|-----------------|---------------|-----------|
|     From        |       Aztec       |      Iden3      |   Starknet  |     Aleo    |
| Compiles to |   IR (ACIR)   |       R1CS      |   Sierra (IR)  |    R1CS   |
|    Supports    | Integrated with UltraPlonk, Extendable to Groth16, Halo2  |   Groth16   |    Stark, FRI |  SNARK-based  |
|    Used By    | Verifier can be deployed on EVM compatible chains  |  Tornado Cash, Polygon Hermez, Dark Forest, Semaphore |   Starknet   |       Aleo      |

**Comment by carlos-israelj** [comment]
hi! @vplasencia @aguzmant103 is this issue still Issues? could I take it? I am part of the PSE 2024 core program

**Comment by vplasencia** [comment]
Hey @cijimenez 👋

Yes, it's still open. Do you want me to assign this issue to you?

**Comment by carlos-israelj** [comment]
Hi @vplasencia 
Yes, of course! Please assign it to me. I would also like to get more details and understand the specific objective of this issue. Thanks!

**Comment by vplasencia** [comment]
Hey @cijimenez! Done. Thanks. Feel free to ask any questions you may have.

The main idea of the issue is to do some research on some similar projects working (or that worked) on similar topics to explain the similarities and differences with Semaphore. The issue description has some ideas but feel free to do some research about it and add other projects too. 

You can decide on the most effective way to compare them, including which parameters to use for the comparison.

**Comment by vplasencia** [comment]
Hey @cijimenez! Thank you very much for the great work! Do you need any help to add the document to the website and documentation?
